test/modules/ModuleRestarter/src/Settings.cs

```diff
@@ -49,9 +47,8 @@ public class Settings
             }
         }
 
-        public static Settings Current => DefaultSettings.Value;
+        internal static Settings Current => DefaultSettings.Value;
 
-        [JsonIgnore]
         public string ServiceClientConnectionString { get; }
```

Comment:
Should these properties be internal as well?

Reply:
same explanation as above